xxhash.h

```diff
@@ -3197,12 +3209,16 @@ enum XXH_VECTOR_TYPE /* fake enum */ {
 #     define XXH_ACC_ALIGN 16
 #  elif XXH_VECTOR == XXH_AVX512  /* avx512 */
 #     define XXH_ACC_ALIGN 64
+#  elif XXH_VECTOR == XXH_SVE   /* sve */
+#     define XXH_ACC_ALIGN 64
```

Comment:
Quick question : 

This variant seems to directly target 512-bit width.

My understanding of `SVE` is that it's more flexible with regards to vector width, compared to Intel's `AVX`.
That being said, the `qemu` documentation implies that there are multiple hardware implementations of `SVE` to expect, from 128 to 512 bits.

Will this vector extension work with all of them, or only specifically on the 512-bit one ?

Reply:
Yes, you're right. SVE ranges from 128-bit to 2048-bit. Should I create a macro to calculate it for SVE?

Reply:
I've configured CI to run code from SVE128 to SVE2048. All of them could work with this XXH_ACC_ALIGN.